Description: add explicit require for minitest
Forwarded: yes, https://github.com/adammck/minitest-stub-const/issues/7
Author: Pirate Praveen <praveen@debian.org>

Index: ruby-minitest-stub-const-0.4/test/test_stub_const.rb
===================================================================
--- ruby-minitest-stub-const-0.4.orig/test/test_stub_const.rb
+++ ruby-minitest-stub-const-0.4/test/test_stub_const.rb
@@ -1,5 +1,6 @@
 require File.expand_path('../../lib/minitest/stub_const', __FILE__)
 require 'minitest/mock'
+require 'minitest/autorun'
 
 module A
   module B
